Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,40 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
+            string name = PromptString("name");
+            string surname = PromptString("surname");
+            int age = PromptInt("age");
+            int weight = PromptInt("weight");
```

Comment:
This is rather of type float. :)

Reply:
What parts are you suggesting that I convert to float? The age and weight? Or just the weight? Or is that not what you are saying?

Reply:
Age is not a float, because we don't really say "I am 25.7 years old". So integer is fine for age.

Reply:
I am still a bit confused because I have age as an "int" there already. Weight is now a float though if that is what you mean.